src/clj/puppetlabs/services/jruby_pool_manager/jruby_core.clj

```diff
@@ -162,6 +167,8 @@
   [config :- {schema/Keyword schema/Any}]
   (-> config
       (update-in [:compile-mode] #(keyword (or % default-jruby-compile-mode)))
+      (update-in [:profiling-mode] #(keyword (or % default-jruby-profiling-mode)))
```

Comment:
Hm I know this is for consistency but I personally think it's fine to just use `:off` here and with compile mode

Reply:
Nitpick that this just made me realize though: if the JRuby setting is 'ProfilerMode', it would be more consistent for our setting to be called :profiler-mode

Reply:
All the docs for the Java functions had `ProfilingMode`, we actually went through and renamed it to that. Did you find the setting called `ProfilerMode` somewhere?

Reply:
It just mentioned that in the docstring above for "default-jruby-profiling-mode", so I guess it was the docstring that was inconsistent?

Reply:
I inlined the default and removed the def with the bad docstring